addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2274,25 +2274,6 @@ options.registry.HeaderNavbar = options.Class.extend({
         this.setTarget(this.$target.closest('#wrapwrap > header'));
     },
 
-    //--------------------------------------------------------------------------
-    // Public
-    //--------------------------------------------------------------------------
-
-    /**
-     * @private
-     */
-    async updateUI() {
-        await this._super(...arguments);
-        // For all header templates except those in the following array, change
-        // the title of the option to "Mobile Alignment" (instead of
-        // "Alignment") because it only impacts the mobile view.
-        if (!["'default'", "'hamburger'", "'sidebar'", "'magazine'", "'hamburger-full'", "'slogan'"]
-                .includes(weUtils.getCSSVariableValue("header-template"))) {
-            this.el.querySelector("[data-name='header_alignment_opt']").title =
-                _t("Mobile Alignment");
-        }
-    },
```

Comment:
I think I'll end up doing a rebase-merge commit instead of a rebase-ff as I saw some mistakes like this one: here is another example of a commit which probably does not make sense on its own. Only removing code means one of your previous commit made that code useless and this is that commit which should contain this removal.

Reply:
I got your point 👍 to fix that :

- I removed `"'magazine'", "'hamburger-full'", "'slogan'"` from the condition in https://github.com/odoo/odoo/pull/119650/commits/85c82c5df73da9263aefd07f55710bf376adb1ec
- Then I squashed the commit with https://github.com/odoo/odoo/pull/119650/commits/89c70544da40dce5420abd5744d77a915db30362 since they are related.